plugin/storage/integration/badgerstore_test.go

```diff
@@ -49,26 +53,22 @@ func (s *BadgerIntegrationStorage) initialize(t *testing.T) {
 	s.CleanUp = s.cleanUp
 
 	s.logger, _ = testutils.NewLogger()
```

Comment:
these two should also move to L64 - these fields are not changing once initialized. Also, this purpose of this function is to initialize the _factory_, not to initialize the _test suite_. So let's keep the code more tidy.